allennlp/modules/token_embedders/elmo_token_embedder.py

```diff
@@ -0,0 +1,219 @@
+
+        self._load_weights()
+
+    @overrides
+    def get_output_dim(self) -> int:
+        return self.output_dim
+
+    @overrides
+    def forward(self, inputs):  # pylint: disable=arguments-differ
+        # Add BOS/EOS
+        mask = ((inputs > 0).sum(dim=-1) > 0).long()
+        character_ids_with_bos_eos, mask_with_bos_eos = add_bos_eos(
+            inputs,
+            mask,
+            Variable(torch.from_numpy(numpy.array(ELMoCharacterMapper.bos_chars))),
+            Variable(torch.from_numpy(numpy.array(ELMoCharacterMapper.eos_chars)))
+        )
+
+        # the character id embedding
+        max_chars_per_token = self._options['char_cnn']['max_characters_per_token']
+        # (batch_size * sequence_length, max_chars_per_token, embed_dim)
+        char_embeds = torch.nn.functional.embedding(
+            character_ids_with_bos_eos.view(-1, max_chars_per_token),
+            self._char_embedding_weights
+        )
+
+        # run convolutions
+        cnn_options = self._options['char_cnn']
+        if cnn_options['activation'] == 'tanh':
+            activation = torch.nn.functional.tanh
+        elif cnn_options['activation'] == 'relu':
+            activation = torch.nn.functional.relu
+        else:
+            raise ConfigurationError("Unknown activation")
+
+        # (batch_size * sequence_length, embed_dim, max_chars_per_token)
+        char_embeds = torch.transpose(char_embeds, 1, 2)
+        convs = []
+        for conv in self._convolutions:
+            convolved = conv(char_embeds)
+            # (batch_size * sequence_length, n_filters for this width)
+            convolved, _ = torch.max(convolved, dim=-1)
+            convolved = activation(convolved)
+            convs.append(convolved)
+
+        # (batch_size * sequence_length, n_filters)
+        ret = torch.cat(convs, dim=-1)
+
+        # apply the highway layers (batch_size * sequence_length, n_filters)
+        ret = self._highways(ret)
+
+        # final projection  (batch_size * sequence_length, embedding_dim)
+        ret = self._projection(ret)
+
+        # reshape to (batch_size, sequence_length, embedding_dim)
+        batch_size, sequence_length, _ = inputs.data.shape
+
+        return {
+            'mask': mask_with_bos_eos,
+            'token_embedding': ret.view(batch_size, sequence_length + 2, -1)
+        }
+
+    def _load_weights(self):
+        self._load_char_embedding()
+        self._load_cnn_weights()
+        self._load_highway()
+        self._load_projection()
+
+    def _load_char_embedding(self):
+        with h5py.File(cached_path(self._weight_file), 'r') as fin:
+            char_embed_weights = fin['char_embed'][...]
+
+        weights = numpy.zeros(
+            (char_embed_weights.shape[0] + 1, char_embed_weights.shape[1]),
+            dtype=DTYPE
+        )
+        weights[1:, :] = char_embed_weights
+
+        self._char_embedding_weights = torch.nn.Parameter(
```

Comment:
This is where the `bad-continuation` is coming from - pylint wants this formatted as : 
```
self._char_embedding_weights = torch.nn.Parameter(torch.FloatTensor(weights),
                                                  requires_grad=False)
```